__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const { TestWatcher } = require('jest');
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+//3
+describe('GET /api/topics task 3',() => {
```

Comment:
I would say that these three test blocks could be combined as they are all testing the response